.github/ISSUE_TEMPLATE/spec_request.md

```diff
@@ -0,0 +1,55 @@
+---
+title: 'Request creation of a new format specification'
+labels: spec-request
+---
+<!--
+A request for a format specification consists of 2 YAML blocks followed by a free-form text description.
+Fill in all the fields possible. Delete fields you cannot populate and all the comments from this template (including this one!).
```

Comment:
"... or, if not possible, do not fill the fields". What's the point in having this sentence?

Asking to delete the comments is (1) ambiguous — should they delete only XML comments, or YAML comments as well?, (2) tons of work.

Reply:
>Asking to delete the comments is (1) ambiguous — should they delete only XML comments, or YAML comments as well?

Every comment orignated from the template, no matter YAML or HTML or of any other formal language ones. Comments there are just to inform a user, so they are just junk in the filled request form. That's why it is asked to remove them. YAML comments especially must be removed, because they stay visible. HTML comments also should be removed because they are junk, but even if not, they affect only the ones seeing the message source code.